code/datums/action.dm

```diff
@@ -126,25 +136,28 @@
 
 /datum/action/item_action/Trigger(attack_self = TRUE) //Maybe we don't want to click the thing itself
 	if(!..())
-		return 0
+		return FALSE
 	if(target && attack_self)
 		var/obj/item/I = target
 		I.ui_action_click(owner, type)
-	return 1
+	return TRUE
 
 /datum/action/item_action/ApplyIcon(obj/screen/movable/action_button/current_button)
+	current_button.overlays.Cut()
```

Comment:
We don't do this anymore.

`cut_overlays`

Reply:
Found about 170 instances of that, so I replaced them all by `cut_overlays`